website/docs/api/spancategorizer.md

```diff
@@ -13,6 +13,9 @@ A span categorizer consists of two parts: a [suggester function](#suggesters)
 that proposes candidate spans, which may or may not overlap, and a labeler model
 that predicts zero or more labels for each candidate.
 
+Predicted spans will be saved in a [`SpanGroup`](/api/spangroup) on the doc.
+Individual span scores can be found in `spangroup.attrs["scores"]`.
```

Comment:
Was not really sure where to put the docs for this so I just put it in this opening section. Is there some convention for this I can follow? 

I looked at other components, but for example the textcat docs never explicity say "the categories are saved in `doc.cats`", though the code explicitly notes that's assigned to.